### PR TITLE
Make tritium-tracing compatible with remoting-3

### PR DESCRIPTION
**Pull request by j-baker**
At the moment it is not compatible - generated dynamic proxies will log into remoting 1 trace logs, which will not be listened to due to package name changes (yes this is rubbish, but will push for a fix in remoting 4).

### CODE REVIEW

File: versions.props [hunk 1]
```diff
@@ -4,7 +4,7 @@ com.google.code.findbugs:* = 3.0.1
 com.google.code.findbugs:jFormatString = 3.0.0
 com.google.guava:* = 18.0
 com.palantir.baseline:* = 0.16.0
-com.palantir.remoting1:* = 1.0.0
+com.palantir.remoting3:* = 3.3.2
```
Comment (schlosna) [inline versions.props]: need 3.3.3

**Comment by schlosna** [comment]
Separate from this, but would really like to pull the tracing stuff out of http-remoting into its own stable versioned thing that tritium & http-remoting both depend upon

**Comment by j-baker** [comment]
Yep! filed https://github.com/palantir/http-remoting/issues/580

File: versions.props (hunk 1 above)
Comment (j-baker) [reply versions.props]: done

**Pull request merged**